superset-frontend/src/dashboard/components/nativeFilters/FiltersConfigModal/FilterTitlePane.tsx

```diff
@@ -104,7 +90,12 @@ const FilterTitlePane: React.FC<Props> = ({
   );
   return (
     <TabsContainer>
-      <StyledHeader>Filters</StyledHeader>
+      <Dropdown overlay={menu} arrow placement="topLeft" trigger={['hover']}>
+        <StyledAddBox>
+          <div data-test="new-dropdown-icon" className="fa fa-plus" />{' '}
+          <span>{t('Add Filters and Dividers')}</span>
```

Comment:
Super nit. It looks like we are using camel case for headers but I don't think we are for the buttons. Should this be `Add filters and dividers` instead?

```suggestion
          <span>{t('Add filters and dividers')}</span>
```

P.S. If you accept this change Cypress will need to be updated as well

Reply:
@kasiazjc What do you think?

Reply:
According to [the rules](https://github.com/apache/superset/wiki/Capitalization-Guidelines) it should be sentence case. If we want to change the rules, we can do that. But in the name of expediency, I'd accept @geido's change and adjust the test accordingly.

Reply:
(i.e. filter and dividers fall under the "common noun" rule, since they're not pages / nav items, which would be capitalized)

Reply:
Done. Thanks for the review!